### ISSUE TITLE
strange char in svg plots

### ISSUE THREAD

**Issue by randy3k** [IRkernel/IRkernel#117]
When I do this in one block

```
options(jupyter.plot_mimetypes = 'image/svg+xml')
plot(1:10)
curve(x^2)
```

The second plot contains weird axises and ticks.
![screen shot 2015-04-29 at 10 21 05 pm](https://cloud.githubusercontent.com/assets/1690993/7406801/0c3bec60-eebe-11e4-81b3-82be320ed7e7.png)

I also note that if I put the `plot` and `curve` in different blocks, it doesn't happen. It also doesn't happen when `image/png` is used instead. 

Mac OSX 10, R 3.2, latest master branch of `IRkernel`, `IRdisplay` and `repr`.

**Comment by flying-sheep** [comment]
probably your browser’s SVG rendering being broken.

firefox’ certainly is.

**Comment by randy3k** [comment]
It still doesn't explain why it works when I separate the two commands.

**Comment by flying-sheep** [comment]
strange stuff :confused:

**Comment by JanSchulz** [comment]
Dupe of #125?

**Comment by flying-sheep** [comment]
i’ll mark things as dupe of this and close this as soon as people confirm that IRkernel/repr#8 fixed it.

**Comment by flying-sheep** [comment]
well this was first, so #125 is a dupe of this.

we should try if it works with `metadata = list('image/svg+xml' = list(isolated = TRUE))`

**Comment by takluyver** [comment]
#128 uses 'isolated' - I hope that will improve this.

**Comment by flying-sheep** [comment]
fixed by  #128

**Comment by rgbkrk** [comment]
Looking great now!

![screenshot 2015-06-03 15 53 43](https://cloud.githubusercontent.com/assets/836375/7971212/b9d4065e-0a08-11e5-910b-6694c2acd3c6.png)
